src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -1402,6 +1486,58 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
     @SuppressWarnings({"Indentation", "unused"})
     private Object[] generateAppointeeNotificationScenarios() {
         return new Object[]{
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Collections.emptyList(),
+                Collections.emptyList(),
+                Arrays.asList("6af62d46-98e5-4ade-aa72-e4a11c56286e"),
+                "no",
+                "no",
+                "0",
+                "0",
+                "1",
+                "Harry Potter"
+            },
+            new Object[]{
+                EVIDENCE_RECEIVED_NOTIFICATION,
+                "paper",
+                Arrays.asList("c5654134-2e13-4541-ac73-334a5b5cdbb6"),
+                Collections.emptyList(),
+                Collections.emptyList(),
+                "yes",
+                "no",
+                "1",
+                "0",
+                "0",
```

Comment:
According to the ticket, regardless of the subscription for the appointee, this letter should be sent to them.